flask/packages.py

```diff
@@ -210,7 +216,9 @@ def upload():
         package_obj = {
             "name": package_name,
             "namespace": namespace_doc["_id"],
-            "description": "Sample Test description",
+            "description": package_data["description"],
```

Comment:
Will this run into exception if there is not any key in `package_data` ?

Reply:
we can handle exceptions , but I think we must make it compulsory for the user to add description.